external_search.py

```diff
@@ -197,24 +197,47 @@ def make_target_age_query(target_age):
                 }
             }
 
-        # These fields have been stemmed non-minimally and shouldn't be used with
-        # fuzzy queries.
-        stemmed_fields = [
+
+        stemmed_query_string_fields = [
+            # These fields have been stemmed.
             'title^4',
             "series^4", 
             'subtitle^3',
             'summary^2',
+
+            # These fields only use the standard analyzer and are closer to the
+            # original text.
+            "classifications.term^2",
+            'author^6',
+            'publisher',
+            'imprint'
         ]
 
-        # Minimally stemmed or standard fields have higher weight since they're closer to the
-        # original text.
-        other_fields = [
-            'author^5',
+        minimal_query_string_fields = [
+            # These fields have been minimally stemmed, so they have
+            # a higher weight than the stemmed fields.
             'title.minimal^5',
-            'series.minimal^5',
-            "subtitle.minimal^4",
-            "summary.minimal^3",
+            "series.minimal^5", 
+            'subtitle.minimal^4',
+            'summary.minimal^3',
+
+            # These fields only use the standard analyzer and are closer to the
+            # original text.
             "classifications.term^3",
+            'author^6',
+            'publisher',
+            'imprint'
+        ]
+
+        fuzzy_fields = [
+            # Only minimal stemming should be used with fuzzy queries.
+            'title.minimal^4',
+            'series.minimal^4',
+            "subtitle.minimal^3",
+            "summary.minimal^2",
+
+            "classifications.term^2",
+            'author^4',
             'publisher',
             'imprint'
         ]
```

Comment:
I broke these up more so I could give the minimal fields higher weight for both non-stemmed and non-fuzzy queries, and so I could give author a higher weight in non-fuzzy queries.